extensions/azurePublish/src/node/deploy.ts

```diff
@@ -275,15 +275,22 @@ export class BotProjectDeploy {
 
   // Upload the zip file to Azure
   // DOCS HERE: https://docs.microsoft.com/en-us/azure/app-service/deploy-zip
-  private async deployZip(token: string, zipPath: string, name: string, env: string, hostname?: string) {
+  private async deployZip(
+    token: string,
+    zipPath: string,
+    name: string,
+    env: string,
+    hostname?: string,
+    scmHostDomain?: string
+  ) {
     this.logger({
       status: BotProjectDeployLoggerType.DEPLOY_INFO,
       message: `Uploading zip file... to ${hostname ? hostname : name + (env ? '-' + env : '')}`,
     });
 
-    const publishEndpoint = `https://${
-      hostname ? hostname : name + (env ? '-' + env : '')
-    }.scm.azurewebsites.net/zipdeploy/?isAsync=true`;
+    const publishEndpoint = `https://${hostname ? hostname : name + (env ? '-' + env : '')}.${
```

Comment:
nit: can we put this logic in a method? Or evaluate each part as variables in lines above? Just a lot of logic to read in a template string!